.python-version

```diff
@@ -1 +1 @@
-3.10.4
+3.12.2
```

Comment:
we could probably update this to 3.12.4 since that seems to be the global version in packer?